client/src/main.rs

```diff
@@ -39,6 +39,12 @@ async fn main_inner() -> Result<(), Error> {
         },
     )?;
 
+    // Simply log the Cargo package version a.k.a version of the client and return early
+    if command_line.version {
+        log::info!("Client version {}", CLIENT_VERSION);
```

Comment:
```suggestion
        log::info!("nimiq-client {}", CLIENT_VERSION);
```
I checked a couple of `--version` flags on my machine and most have the output `<PROGRAM> <VERSION>`.